code/modules/cooking/plates.dm

```diff
@@ -186,3 +186,43 @@ Plates that can hold your cooking stuff
 	desc = "A small ornamental cauldron used as an altar by the worshippers of Zhukamir, the Ma'ta'ke deity of agriculture and cooking."
 	icon = 'icons/obj/tajara_items.dmi'
 	icon_state = "zhukamir"
+
+/obj/item/reagent_containers/bowl/gravy_boat
+	name = "gravy boat"
+	desc = "Let's sail the seas of deliciousness!"
+	icon_state = "gravy"
+	amount_per_transfer_from_this = 5
+	possible_transfer_amounts = list(5, 10, 30)
+	w_class = WEIGHT_CLASS_SMALL
+	volume = 40
+	force = 14
+	atom_flags = ATOM_FLAG_OPEN_CONTAINER
+	item_flags = ITEM_FLAG_NO_BLUDGEON
+	drop_sound = 'sound/items/drop/glass.ogg'
+	pickup_sound = 'sound/items/pickup/glass.ogg'
+
+/obj/item/reagent_containers/bowl/gravy_boat/afterattack(var/obj/target, var/mob/user, var/flag)
+	if(!target.is_open_container() || !flag)
+		return ..(target, user, flag)
+	if(reagents.total_volume)
+		if(!target.reagents || !REAGENTS_FREE_SPACE(target.reagents))
+			to_chat(user, SPAN_NOTICE("[target] is full."))
+			return TRUE
+		var/trans = reagents.trans_to(target, amount_per_transfer_from_this) //sprinkling reagents on generic non-mobs
+		user.visible_message(
+			"<b>[user]</b> pours onto [target] from [src].",
+			SPAN_NOTICE("You transfer [trans] units of the solution.")
+		)
+		playsound(user.loc, /singleton/sound_category/generic_pour_sound, 10, 1)
```

Comment:
No magic numbers

Reply:
idk what magic numbers are i copied this code from somewhere else ._.

Reply:
https://en.wikipedia.org/wiki/Magic_number_(programming)

```
playsound(user.loc, /singleton/sound_category/generic_pour_sound, 10, TRUE)
```